src/to-have-style.js

```diff
@@ -50,7 +50,7 @@ function expectedDiff(diffFn, expected, computedStyles) {
     )
   const diffOutput = diffFn(printoutStyles(expected), printoutStyles(received))
   // Remove the "+ Received" annotation because this is a one-way diff
-  return diffOutput.replace(`${chalk.red('+ Received')}\n`, '')
+  return diffOutput.replace(`${picocolors.red('+ Received')}\n`, '')
```

Comment:
The smaller and faster "drop-in" replacement for `chalk` is [ansis](https://github.com/webdiscus/ansis).

Ansis is one of the smallest packages, see please [Compare package sizes](https://github.com/es-tooling/ecosystem-cleanup/issues/132#compare-size) of most popular libs.

Unpacked Size: 
- [chalk](https://www.npmjs.com/package/chalk) (43.7 kB)
- [ansis](https://www.npmjs.com/package/ansis) (11.3 kB) - with the same functionality as chalk
- [picocolors](https://www.npmjs.com/package/picocolors) (6.37 kB)

Ansis supports the [nested template strings](https://github.com/es-tooling/ecosystem-cleanup/issues/132#nested-syntax), but [chalk and picocolors](https://github.com/webdiscus/ansis#compare) - no. 

So that you can write shorter and more readable code:

```js
const { red, blue } = require('ansis');

diffOutput.replace(`${red`+ Received`}\n`, '')

// as an example for nested template string:
red`contains chained ${blue.bold`sub`} strings`;
```